torch/csrc/distributed/rpc/process_group_agent.cpp

```diff
@@ -285,7 +284,7 @@ std::shared_ptr<FutureMessage> ProcessGroupAgent::send(
     // millisecond level precision of when request started.
     auto futureStartTime = std::chrono::steady_clock::now();
     // Prepare endTime from timeout.
-    auto timeout = rpcTimeout_.load();
+    auto timeout = rpcTimeout;
```

Comment:
why do we need to make a copy here? As the timeout is now per-send, can we use it as-is?